### PR TITLE
Added path formatting for path in mounts

### PR DESCRIPTION
**Pull request by SaswatB**

### CODE REVIEW

File: pkg/server/container_create_windows.go [hunk 1]
```diff
@@ -279,9 +280,22 @@ func (c *criService) generateContainerSpec(id string, sandboxID string, sandboxP
 
 	// Add OCI Mounts
 	for _, m := range config.GetMounts() {
+
+		//normalize the format of the container path
+		var formattedDestination string
```
Comment (jterry75) [inline pkg/server/container_create_windows.go]: Do we need to do this for `m.HostPath` as well? Seems like Kube could also pass `c:/test/test` as a path for the `HostPath`?

**Comment by jterry75** [comment]
Looks awesome. 1 comment

File: pkg/server/container_create_windows.go (hunk 1 above)
Comment (SaswatB) [reply pkg/server/container_create_windows.go]: I think kubelet handles host path correctly, but better safe than sorry I guess.

**Comment by jterry75** [comment]
LGTM

**Pull request merged**